<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CParameterDlgSettings</class>
 <widget class="QDialog" name="CParameterDlgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset theme="system-settings">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="windowTitle">
      <string>Settings</string>
     </property>
     <property name="windowIcon">
      <iconset>
       <normaloff>:/image/Option</normaloff>:/image/Option</iconset>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="General">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="cbFavoriteDoubleEdit">
         <property name="text">
          <string>Favorite: select it then double node edit, other connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Recent open file max count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbRecentMenuMaxCount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Shot screen</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbShotRemoteDesktop">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Select shot remote desktop, otherwise shot window</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QRadioButton" name="rbShotScreenNoAction">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>No action</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbShotScreenOpenFolder">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Open folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbShotScreenOpenFile">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Open file</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Save Path:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leShotScreenSavepath"/>
            </item>
            <item>
             <widget class="QPushButton" name="pbScreenShotBrowser">
              <property name="text">
               <string>Browser(&amp;B)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbResumLasterClose">
         <property name="text">
          <string>Resume the connections when it was last closed at startup</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="cbMessageBoxDisplayInfo">
         <property name="text">
          <string>Use message box to display error information</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main window</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Tab view</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Position</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QRadioButton" name="rbNorth">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>North</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="rbSouth">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>South</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QRadioButton" name="rbWest">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>West</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QRadioButton" name="rbEast">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>East</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="cbTabIcon">
              <property name="text">
               <string>Icon</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbTabToolTip">
              <property name="text">
               <string>ToolTip</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="gbSystemTrayIconContextMenu">
         <property name="title">
          <string>System tray icon context menu</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QRadioButton" name="rbSystemTrayIconRemote">
            <property name="text">
             <string>Remote</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="rbMenuBar">
            <property name="text">
             <string>Main menu bar</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbSystemTrayIconNo">
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="rbSystemTrayIconRecentOpen">
            <property name="text">
             <string>Recent open</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>66</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbReciverShortCut">
         <property name="text">
          <string>Main window receiver short cut key</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="cbShowSystemTrayIcon">
         <property name="text">
          <string>Enable system tray icon</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbSaveMainWindowStatus">
         <property name="text">
          <string>Save main window status</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>Ok(&amp;O)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNo">
       <property name="text">
        <string>No(&amp;N)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>sbRecentMenuMaxCount</tabstop>
  <tabstop>cbResumLasterClose</tabstop>
  <tabstop>cbFavoriteDoubleEdit</tabstop>
  <tabstop>cbShotRemoteDesktop</tabstop>
  <tabstop>rbShotScreenNoAction</tabstop>
  <tabstop>rbShotScreenOpenFolder</tabstop>
  <tabstop>rbShotScreenOpenFile</tabstop>
  <tabstop>leShotScreenSavepath</tabstop>
  <tabstop>pbScreenShotBrowser</tabstop>
  <tabstop>cbReciverShortCut</tabstop>
  <tabstop>cbSaveMainWindowStatus</tabstop>
  <tabstop>cbShowSystemTrayIcon</tabstop>
  <tabstop>rbSystemTrayIconNo</tabstop>
  <tabstop>rbMenuBar</tabstop>
  <tabstop>rbSystemTrayIconRemote</tabstop>
  <tabstop>rbSystemTrayIconRecentOpen</tabstop>
  <tabstop>rbNorth</tabstop>
  <tabstop>rbSouth</tabstop>
  <tabstop>rbWest</tabstop>
  <tabstop>rbEast</tabstop>
  <tabstop>cbTabIcon</tabstop>
  <tabstop>cbTabToolTip</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbNo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
